sentencepiece.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: sentencepiece
+  version: 0.2.0
+  epoch: 0
+  description: "SentencePiece is an unsupervised text tokenizer and detokenizer"
+  copyright:
+    - license: Apache-2.0
+
+vars:
+  pypi-package: sentencepiece
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - cmake
+      - py3-supported-pip
+      - py3-supported-python-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/google/sentencepiece.git
+      tag: 'v${{package.version}}'
+      expected-commit: 17d7580d6407802f85855d2cc9190634e2c95624
+
+  - uses: cmake/configure
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package]} installed for python${{range.key}}
+    pipeline:
+      - runs: python${{range.key}} -m pip install ./python "--root=${{targets.subpkgdir}}" --prefix=/usr
+      - uses: strip
+    dependencies:
+      runtime:
+        - py3-${{vars.pypi-package}}
```

Comment:
the `py3-${{vars.pypi-package}}` dependency looks incorrect

The py3-sentencepiece virtual should be handled by the provides below. You allso need to use provider-priority set to `${{range.value}}` for these subpackages.